app.js

```diff
@@ -13,3 +29,129 @@ function openMenu() {
   menu.style.backgroundColor = '#384af5';
   menu.style.display = 'block';
 }
+
+function popUpProject(popUpdata) {
+  const popUpItem = `<div id="popUp-view">
+     <div class="popUp-card">
+         <img src="images/close.png" alt="close" id="popUp-Close" onclick="popupClose();">
+         <h2 class="card-title">${popUpdata.name}</h2>
+         <div class="work-detail-box">
+             <ul>
+                 <li class=" work-detail canopy">CANOPY </li>
+                 <li class="work-detail work-text">Back End Dev </li>
+                 <li class="work-detail work-text">2015</li>
+             </ul>
+         </div>
+         <div class="card-image">
+             <img class="work-snapshot" src=${popUpdata.imgUrl}
+                 alt="snapshot">
+         </div>
+         <div class="work">
+             <p class="work-description">${popUpdata.detailDescription}</p>
+             <div>
+                 <div class="work-detail-box">
+                     <ul>
+                         <li class="chips">html</li>
+                         <li class="chips">css</li>
+                         <li class="chips">javascript</li>
+                     </ul>
+                 </div>
+                 <div id="work-buttons">
+                     <div class="work-button"><a class="project-button" href=${popUpdata.liveDemoLink}>See Live <img
+                         src="images/live1.png" alt="live" id="live"></a>
+                     </div>
+                     <div class="work-button"><a class="project-button" href=${popUpdata.github}>See Source <img
+                                 src="images/github.png" alt="github" id="pop-github"></a></div>
+                 </div>
+  
+             </div>
+  
+         </div>
+     </div>
+  </div>`;
+  workPopUpCard.innerHTML = popUpItem;
+}
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Kindly remove all the eslint disabled rules here.

Reply:
These are unused variables but used at run time. how am i suppose to clear eslint test?